src/webapp/components/dataset-wizard/SummaryDataSet.tsx

```diff
@@ -64,47 +66,78 @@ export const SummaryList = React.memo((props: { dataSet: DataSet }) => {
                     value={selectedRegions.map(region => region.name).join(", ")}
                 />
 
-                {missingSelectedCompanion.length > 0 && (
-                    <SummaryItem label={i18n.t("Companion Indicators")} value="" />
+                {missingSelectedCompanion && (
+                    <>
+                        <SummaryItem label={i18n.t("Companion Indicators")} value="" />
+                        <MissingCompanionAlert
+                            indicator={missingSelectedCompanion}
+                            requiredScope={Indicator.getIndicatorScope(
+                                dataSet.project?.startDate
+                            )}
+                        />
+                    </>
                 )}
-
-                {missingSelectedCompanion.map(indicator => (
-                    <MissingCompanionAlert indicator={indicator} key={indicator.id} />
-                ))}
             </ul>
         </Grid>
     );
 });
 
-const MissingCompanionAlert = React.memo((props: { indicator: MissingCompanionIndicator }) => {
-    const { indicator } = props;
-
-    const outcomeMessage = indicator.outcomeMessage
-        ? i18n.t("Outcome companion rule not satisfied: {{rule}}", {
-              rule: indicator.outcomeMessage,
-              nsSeparator: false,
-              interpolation: { escapeValue: false },
-          })
-        : undefined;
-
-    const outputMessage = indicator.outputMessage
-        ? i18n.t("Output companion rule not satisfied: {{rule}}", {
-              rule: indicator.outputMessage,
-              nsSeparator: false,
-              interpolation: { escapeValue: false },
-          })
-        : undefined;
-
-    const messages = _([outputMessage, outcomeMessage]).compact().value();
-
-    return (
-        <MissingCompanionAlertContainer>
-            {messages.map(message => (
-                <SummaryItem key={message} label={indicator.code} value={message} />
-            ))}
-        </MissingCompanionAlertContainer>
-    );
-});
+const MissingCompanionAlert = React.memo(
+    (props: { indicator: MissingCompanionIndicator; requiredScope: string }) => {
+        const { indicator, requiredScope } = props;
+
+        const scopes = indicator.keys();
+
+        const scopeText = React.useCallback(
+            (scope: string) => {
+                return scope === requiredScope && scope !== "default" ? (
+                    <strong>
+                        {scope}:{i18n.t("The companion rules for this year need to be satisfied")}
+                    </strong>
+                ) : (
+                    scope
```

Comment:
Not a problem, but can we put a tag for the scope? 

`<p>scope</p>`. 

`span` also works if you need an inline element.